package.json

```diff
@@ -1,37 +1,55 @@
 {
   "name": "react-gmp",
   "version": "0.1.0",
-  "private": true,
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "DEV": "cross-env NODE_ENV=development webpack-dev-server",
+    "PROD": "cross-env NODE_ENV=production webpack --config webpack.config.js",
+    "start": "npx webpack-dev-server --mode development",
+    "build": "npx webpack --mode production",
+    "lint": "eslint --fix",
+    "pretty": "prettier --write .",
+    "prepare": "husky install"
+  },
   "dependencies": {
-    "@testing-library/jest-dom": "^5.16.4",
-    "@testing-library/react": "^13.2.0",
-    "@testing-library/user-event": "^13.5.0",
+    "bootstrap": "^5.1.3",
```

Comment:
if you want to use bootstrap here, use https://react-bootstrap.github.io

Reply:
ok

Reply:
done